### PR TITLE
Update rocDecode for ROCm 6.1.2 release

### PR DESCRIPTION
**Pull request by AryanSalmanpour**

### CODE REVIEW

**Review by kiritigowda — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ updates:
     open-pull-requests-limit: 10
     schedule:
       interval: "daily"
-    target-branch: "master"
+    target-branch: "develop"
```
Comment (rrawther) [inline .github/dependabot.yml:13]: why is this changed to develop?

Comment (AryanSalmanpour) [inline .github/dependabot.yml:13]: @rrawther  because there is no master branch for rocDecode. The above change is for building the doc based on the develop branch.

**Review by rrawther — APPROVED** [review]
